### PR TITLE
Update manifest.json

### PR DESCRIPTION
**Pull request by Kallecrafter**
Fixed any json errors
Add a new Domain
Add a GameMode
Spelling mistake fixed
textcolor changed by Brand
Add Website to Socials

## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by Kallecrafter** [comment]
Its newest pull!

File: minecraft_servers/craftergang/manifest.json [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
   "direct_ip": "craftergang.de",
   "server_wildcards": [
+    "craftergang.net",
     "citybuild.craftergang.de"
   ],
   "supported_languages": [
     "de"
   ],
   "social": {
+    "web": "https://sites.google.com/view/craftergang",
```
Comment (DoJapHD) [inline minecraft_servers/craftergang/manifest.json:13]: Can't be Goggle site, either replace with real website or remove

**Comment by Kallecrafter** [comment]
{
  "server_name": "craftergang",
  "nice_name": "Craftergang",
  "direct_ip": "craftergang.de",
  "server_wildcards": [
    "craftergang.net",
    "citybuild.craftergang.de"
  ],
  "supported_languages": [
    "de"
  ],
  "social": {
    "discord": "https://discord.gg/ThCt8Pq",
    "teamspeak": "craftergang.de",
    "tiktok": "craftergang",
    "youtube": "https://www.youtube.com/@craftergang",
    "instagram": "craftergang"
  },
  "gamemodes": {
    "citybuild": {
      "name": "CityBuild",
      "color": "#0d6914",
      "versions": "1.8<1.14.4"
    },
    "ffa": {
      "name": "FFA",
      "color": "#ed0c26",
      "versions": "1.8<1.14.4"
    },
    "build": {
      "name": "BuildFFA",
      "color": "#e8a507",
      "versions": "1.8<1.14.4"
    },
    "bedwars2x1": {
      "name": "BedWars2x1",
      "color": "#e3bcc1",
      "versions": "1.8<1.14.4"
    },
    "bedwars2x2": {
      "name": "BedWars2x2",
      "color": "#e3bcc1",
      "versions": "1.8<1.14.4"
    },
    "mlgrush": {
      "name": "MLGRush",
      "color": "#de0b39",
      "versions": "1.8<1.14.4"
    },
    "welt-untergang": {
      "name": "Welt-Untergang",
      "color": "#0bde19",
      "versions": "1.8<1.14.4"
    },
    "onehit": {
      "name": "OneHit",
      "color": "#f705ab",
      "versions": "1.8<1.14.4"
    },
    "skypvp": {
      "name": "Old-SkyPvP",
      "color": "#f7df05",
      "versions": "1.8<1.14.4"
    },
    "survival": {
      "name": "Survival",
      "color": "#0d6914",
      "versions": "1.12<1.14.4"
    }
  },
  "brand": {
    "primary": "#497cb1",
    "background": "#245587",
    "text": "#4646cf"
  },
  "location": {
    "city": "Frankfurt",
    "country": "Germany",
    "country_code": "DE"
  },
  "yt_trailer": "7O9dxRuZ4BQ"
}

**Comment by Kallecrafter** [comment]
Close - new one

**Pull request closed without merging**